kafka/secrets.enc.yaml

```diff
@@ -1,171 +1,172 @@
 dh_kafka_git_branch: production
 dh_kafka_git_repo: https://github.com/AICoE/internal-data-hub.git
```

Comment:
Not part of the changes, but this needs to be updated right?

Reply:
Yeah makes sense
I will raise another MR for it